calicoctl/Dockerfile.calicoctl.build

```diff
@@ -1,8 +0,0 @@
-FROM golang:1.7
```

Comment:
Were these files completely unused?  Can't see anywhere that you removed them from the Makefile.

Reply:
Completely. It surprised me, but there you go. I think that they are pre-`go-build`.